app/res/layout/custom_alert_dialog.xml

```diff
@@ -83,7 +83,7 @@
 
                     android:layout_toLeftOf="@id/neutral_button"
 
-                    android:layout_marginRight="@dimen/standard_spacer"
+                    android:layout_marginRight="@dimen/standard_spacer_double"
```

Comment:
I actually didn't mean to leave this change in, because it didn't make any noticeable difference visually. Just reverted